applications/machine_learning/README.rst

```diff
@@ -205,6 +205,28 @@ Check the sample Requirements section for the list of supported development kits
 The sample is used to receive data over NUS and forward it to the host computer over UART.
 See `Testing with the Thingy:52`_ for how to test this solution.
 
+.. _nrf_machine_learning_app_requirements_build_types:
+
+nRF Machine Learning build types
+================================
+
+The nRF Machine Learning application does not use a single :file:`prj.conf` file.
```

Comment:
> does not use a single :file:`prj.conf` file
instead of using various overlays the is single configuration file for a build type. It is just not named prj.conf